### PR TITLE
New Documentation about this project and titles modifications

### PR DESCRIPTION
**Pull request by mauricioromagnollo**
- Changed the title: 
```html 
<title>Document</title> to <title>URL Shortener</title> 
``` 
- Changed the h1:
```html
<h1>URL Shrinker</h1> to <h1>URL Shortener</h1>. 
```
I think this make more sense because the project name;

- Add `.vscode/` inside the `.gitignore` file;

- Added new documentation about this project, for make easy understand and for link the technologies, making easy to access the technologies documentation.

### CODE REVIEW

File: views/index.ejs [hunk 1]
```diff
@@ -5,11 +5,11 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
-  <title>Document</title>
+  <title>URL Shortener</title>
 </head>
 <body>
   <div class="container">
-    <h1>URL Shrinker</h1>
+    <h1>URL Shortener</h1>
```
Comment (WebDevSimplified) [inline views/index.ejs:12]: I do not like to change the files from the course after creating the video so could you remove the changes to the index.ejs file.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## **Using The Project**
+
+If you got the project from GitHub, open the terminal inside the project folder and install all [package.json](./package.json) dependencies with this command:
+
+```bash
+$ npm install
+```
+
+For run the server, use this command:
```
Comment (WebDevSimplified) [inline README.md]: Small grammatical change.
```suggestion
To run the server, use this command:
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+```bash
+$ npm run devStart
+```
+
+Now, open your browser and type: `https://localhost:5000`
```
Comment (WebDevSimplified) [inline README.md]: Need to remove `https://`
```suggestion
Now, open your browser and type: `localhost:5000`
```

File: .gitignore [hunk 4]
```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+.vscode/
```
Comment (WebDevSimplified) [inline .gitignore:2]: I do not like to change the files from the course after creating the video so could you remove the changes from this file.

**Review by WebDevSimplified — CHANGES_REQUESTED** [review]
I really like the changes you made to the Readme in this project. Thank you for taking the time to do that for me. I made a few comments on some grammatical changes as well as removing the changes to files directly used in the video for the course.

File: views/index.ejs (hunk 1 above)
Comment (mauricioromagnollo) [reply views/index.ejs:12]: I understand. I thinked about this when I made the modifications. Sorry about grammar, I am brazilian and my english is not good yet. I liked the project and wanted to expand it. Maybe you should pass to your Git Hub account and separate from that account where you post the course codes. This way, you will also allow your subscribers to submit ideas for the projects you post on the channel. It's just an idea, if you think it's valid send me your Git Hub where you will post these projects as Open Source, I like to contribute. Anyway, congratulations to your videos.

**Pull request closed without merging**